src/sentry/migrations/0385_backfill_user_avatars.py

```diff
@@ -0,0 +1,46 @@
+# Generated by Django 2.2.28 on 2023-01-31 20:37
+
+from django.db import migrations
+
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def backfill_user_avatar(apps, schema_editor):
+    User = apps.get_model("sentry", "User")
+    # UserAvatar = apps.get_model("sentry", "UserAvagtar")
+
+    for user in RangeQuerySetWrapperWithProgressBar(User.objects.all()):
+        avatar = user.avatar.first()
+        if avatar is None:
+            continue
+        user.avatar_type = avatar.avatar_type
+        if avatar.avatar_type == 1:
+            user.avatar_url = f"https://sentry.io/avatar/{avatar.ident}/"
```

Comment:
This migration would run for self hosted users. This new value might not work.

Reply:
@dashed is right. We should use `system.region-api-url-template` or `system.url-prefix` from options here instead as the base URL. The `system.region-api-url-template` requires the current region value though.

Using `generate_region_url()` to get the base URL is likely the better option here.